### DISCUSSION TITLE
How to turn of advance to next file when using multi selection flag?

### DISCUSSION THREAD

**Discussion by hellen-dorandt89** [junegunn/fzf#3121] [Q&A] [answered]
First of all, well done for this brilliant tool! love it

In multi select (`-m`) when I make a selection the active line/selection automatically moves down by one item, can I turn this off? 
Am on windows, Thanks 

![](https://i.imgur.com/sEucDCi.gif)

**Comment by tg0h** [comment, marked as answer]
looking at the man page, ctrl+i (tab) is mapped to toggle+down... which selects the item and moves down

try mapping a key to toggle instead ... eg 
`fzf --bind='ctrl-i:toggle'`

**Reply by hellen-dorandt89** [reply]
Hey that did it, thanks allot, before I posted my question I searched around and did not find an overal view of the parameters available. Could you share a link to this map page you speak of?

**Reply by tg0h** [reply]
in your terminal, type `man fzf`

<img width="960" alt="Screenshot 2023-01-12 at 08 04 57 - kitty@2x" src="https://user-images.githubusercontent.com/7954426/211944449-3be50926-c078-4389-abbd-c926a79d9bbf.png">

or you can check out https://www.mankier.com/1/fzf

I see that you're on a windows ... not sure if man fzf works in windows heh

**Reply by hellen-dorandt89** [reply]
Sweet! suddenly I feel better about my second class citizen dispostion  :)
Thanks!
